test_suite/package/test_otel.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+import logging
+import time
+import json
+
+from test_suite.generic.test_generic import TestsSubscriptionManager
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def check_instance_status(request, instance_data, host):
+    instance_id = instance_data['instance_id']
+    command_to_run = [
+        'aws', 'ec2', 'describe-instance-status',
+        '--instance-ids', instance_id
+    ]
+
+    while True:
+        command_output = host.backend.run_local(" ".join(command_to_run)).stdout
+        cmd_output = json.loads(command_output)
+
+        instance_status = cmd_output["InstanceStatuses"][0]["InstanceStatus"]["Status"]
+        system_status = cmd_output["InstanceStatuses"][0]["SystemStatus"]["Status"]
+
+        if instance_status == "ok" and system_status == "ok":
+            break
+        logger.info("Instance status is not 'passed' yet. Waiting...")
+        time.sleep(5)
+
+
+@pytest.fixture()
+def modify_iam_role(request, instance_data, host):
+    instance_id = instance_data['instance_id']
+    iam_role_name = "CloudWatchAgentServerRole_2"
+
+    command_to_run = [
+        'aws', 'ec2', 'associate-iam-instance-profile',
+        '--instance-id', instance_id,
+        '--region', 'us-west-1',
+        '--iam-instance-profile', 'Name="{}"'.format(iam_role_name)
+    ]
+    modify_iam_role_cmd = ' '.join(command_to_run)
+
+    assert host.backend.run_local(modify_iam_role_cmd), 'faild to update iam role'
+
+
+@pytest.fixture()
+def install_packages(request, host):
+    class_instance = request.node.cls
+    install_cmd = f'yum install -y  {class_instance.package_name} --nogpgcheck'
+    repo_path = ('https://copr.fedorainfracloud.org/coprs/miyunari'
+                 '/redhat-opentelemetry-collector/repo/rhel-9/miyunari-redhat-opentelemetry-collector-rhel-9.repo')
+    with host.sudo():
+        assert host.run_test(f'yum-config-manager --add-repo {repo_path}')
+        assert host.run_test(install_cmd), f'Failed to install the package {class_instance.package_name}'
+        output = host.check_output("rpm -qa | grep opentelemetry*")
+        print(output)
+
+
+@pytest.fixture()
+def start_service(request, host):
+    class_instance = request.node.cls
+    start_enable_service = (f'systemctl start {class_instance.package_name} && '
+                            f'systemctl enable {class_instance.package_name}')
+    is_active = (f'systemctl is-active {class_instance.package_name}')
+
+    with host.sudo():
+        assert host.run(start_enable_service).succeeded, (f'Failed to start the service {class_instance.package_name}')
+        assert host.run(is_active).succeeded, (f'Failed to activate the service {class_instance.package_name}')
+
+    def finalizer():
+        logger.info(f'Removing the package {class_instance.package_name}')
```

Comment:
I like this approach, however I think we need to introduce a logging mechanism in our ticket dedicated for that and make it standard across our project and test suites.

So, for now let's try to be consistent with the current implementations and use our own lib functions.

I suggest to use `test_lib.print_host_command_output()` to output the command and its result.
I also suggest to use `console_lib.print_divider()` to print a "section" in your test case outputs.
you can then simply call `print()` function to print more information inside each section.

See how it has been implemented in this test case as an example: https://github.com/osbuild/cloud-image-val/blob/144f69c99a28c173911fa59abc4890ca06df1b94/test_suite/package/test_efs_utils.py#L33

```suggestion
        console_lib.print_divider(f'Removing the package {class_instance.package_name}')
```